java/ql/src/Likely Bugs/Serialization/NonSerializableComparator.qhelp

```diff
@@ -46,10 +46,10 @@ it does implement <code>Serializable</code>.
 
 
 <li>
-Java API Documentation:
-<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Comparator.html">Comparator</a>,
-<a href="http://docs.oracle.com/javase/6/docs/api/java/io/ObjectOutputStream.html">ObjectOutputStream</a>,
-<a href="http://docs.oracle.com/javase/6/docs/api/java/io/Serializable.html">Serializable</a>.
+Java API Specification:
+<a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Comparator.html">Comparator</a>,
+<a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/ObjectOutputStream.html">ObjectOutputStream</a>,
+<a href=https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/Serializable.html">Serializable</a>.
```

Comment:
```suggestion
<a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/Serializable.html">Serializable</a>.
```

Reply:
Thanks! I have added this change to the "Update Java documentation links to Java 11" commit.